### ISSUE TITLE
Not decompiling correctly

### ISSUE THREAD

**Issue by CyberDie22** [ShaneBeeStudios/McDeob#4]
Unless I'm doing something incorrectly, the decompiled source code doesn't contain a lot of classes required for Minecraft to run.

**Comment by CyberDie22** [comment]
![image](https://user-images.githubusercontent.com/65131969/102571860-7e6d4480-409f-11eb-966a-7780570d9ad8.png)
In this screenshot, the CommandDispatcher class doesn't exist, although it might not have decompiled/deobfuscated correctly

**Comment by ShaneBeee** [comment]
Im going to have to add this to Spigot page, but the decompiled classes will not be able to be built.
I believe the remapper/decompiler I am using, skips classes that are not mapped, I believe this was done to shorten the length of decompiling, as it takes a while.

You can easily make a maven project, and import the missing parts yourself, but do keep in mind, the decompiler and remapper break a lot of the classes, A LOT! 
Long story short, this program is more for your own enjoyment, to get an idea of how the Minecraft code is written, you will not be able to rebuilt the project, without spending a LOT of hours cleaning up all the decompile errors. 

In the future I may look into a better remapper, to alleviate some of the issues here, but for the time being, that is not the point of this app.
